### ISSUE TITLE
I want to which object is in contact with which object using mjContact, etc.

### ISSUE THREAD

**Issue by miyukin73** [deepmind/mujoco#680]
Hello, 

I want to know which object is in contact with which object using mjContact, etc.

If there is contact between objects as shown in the figure below (yellow area), 
is it possible to know which object is in contact with which object, 
such as **geom name**, **id number,** etc. using `mjContact`, etc.?

![image](https://user-images.githubusercontent.com/63595993/210000113-b0fbb096-7f23-4799-84b5-6af2620d65cd.png)
![image](https://user-images.githubusercontent.com/63595993/210000206-6c35e854-7333-4212-aa62-b66afd660e27.png)

Thank you!

**Comment by v-r-a** [comment]
See if you find [this](https://github.com/deepmind/mujoco/issues/132#issuecomment-1229432247) useful.

**Comment by miyukin73** [comment]
Hi, @v-r-a 

Thanks to your reply, I was able to solve this problem!

But I would like to ask one more question, if possible, and I am very sorry that it is a basic question.

I would like to obtain the contact force applied to an object.
To do this, I used `cfrc_ext` and others, but the value was always 0, even though the objects are in contact with each other.

On the other hand, using `qfrc_smooth`, `qacc_smooth` and `qfrc_constraint`, I was able to obtain non-zero values.

What is the difference between these two?

Also, are we getting the correct contact forces between objects using `qfrc_smooth`, `qacc_smooth` and `qfrc_constraint` ?

Thank you very much for your time, and we would greatly appreciate your response.
Thank you in advance.
